### PR TITLE
Update MIPoPS GUI ideas

### PR DESCRIPTION
**Pull request by privatezero**

### CODE REVIEW

**Comment by libbyhopfauf** [comment]
Here are the logos:
vrecord logo audio
vrecord logo documentation
vrecord logo edit
vrecord logo help
vrecord logo logs
vrecord logo playback
vrecord logo

**Comment by privatezero** [comment]
Alright! Let's wait to merge this until tomorrow and I can integrate the new ones.

**Comment by libbyhopfauf** [comment]
Sounds like a plan!

File: vrecord [hunk 1]
```diff
@@ -79,35 +79,64 @@ gui_conf="
...
+    open https://github.com/amiaopensource/vrecord#vrecord-documentation && _master_gui
+elif [[ "${runtype}" = "GUI" ]] ; then
+    echo "Exiting Vrecord. Goodbye!" && exit 0
 fi
-if [[ "${runtype}" = "GUI" ]] ; then
-    echo "Exiting VRecord. Goodbye!" && exit 0
-fi
-rm $pashua_configfile
+
+rm $pashua_configfile 2> /dev/null
```
Comment (privatezero) [inline vrecord]: made change in https://github.com/amiaopensource/vrecord/pull/121/commits/4d69439724d776f892f5e8a3af351e6f323976ad

**Comment by privatezero** [comment]
Now we just need @libbyhopfauf to design another uber logo to represent the entire project!

**Comment by libbyhopfauf** [comment]
I'm on it!

**Pull request merged**